### DISCUSSION TITLE
Get Started | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#414] [Q&A]
# Get Started | Seeed Studio Wiki

Get Started with Wio-WM1110 Dev Kit

https://wiki.seeedstudio.com/Wio-WM1110_Dev_Kit_Server_Configuration/

**Comment by matsujirushi** [comment]
I got these error message when import the modem.json in Node-RED.

![image](https://github.com/Seeed-Studio/wiki-documents/assets/8079404/8f7c7ee0-e3c0-4d2f-ac2d-9c5b433202dc)
![image](https://github.com/Seeed-Studio/wiki-documents/assets/8079404/82ef9d8f-7c16-44cc-800b-90749e73b4f0)


How can I resolve this?

**Reply by matsujirushi** [reply]
We need to install this package. 😊

![image](https://github.com/Seeed-Studio/wiki-documents/assets/8079404/38b664ac-214d-4d84-a6a6-f807485e2217)
![image](https://github.com/Seeed-Studio/wiki-documents/assets/8079404/2b6a8f9e-940a-4693-ac56-32eed3c49fdf)

**Reply by Jessie219-web** [reply]
Hi, thanks for your feedback, will update the Wiki

**Comment by TejaswinSingh** [comment]
Does this J-Link Debug Programmer comes with the kit? And is it necessary? Sorry, I'm new too this stuff.

**Reply by pjeutr** [reply]
No the J-link Debug Programmer is not in the kit, they can be expensive,
I was under the impression the board could be programmed through usb, but fail to run the blinky project with only usb.
Tried this in ubuntu and osx, if it is possible to program through usb. I would like to hear how!
See also https://forum.seeedstudio.com/t/usb-j-link-interface-for-wio-wm1110-dev-kit/271172

**Comment by quanguyen** [comment]
After adding Wio devkit successfully, I cannot configure it.

Details:

From Device page, I tap on the Wio Tracker 1110 Dev Board icon to go to Map page, then tap on the Settings icon at the upper right corner, and select “Device Bluetooth Configuration”.

On the next page, SenseCAP Mate app cannot discover the Board, scanning icon just spins around.
Please note that: I did allow Bluetooth persmission and all kinds of location permission on my SenseCAP Mate iphone app.

How do I get the Board enter Configuration mode, just like pressing & holding the button on T1000 tracker?

**Comment by vinayyn7** [comment]
INFO: Modem Initialization
Lr1mac context : Try recover NVM
WARN: Fail to recover
WARN: No valid lr1mac context --> Use default value
WARN: No valid DevNonce in NVM, use default (0)
 DevNonce = 0
 JoinNonce = 0xff ff ff, NetID = 0xff ff ff
 NbOfReset = 1
 Region = EU868
ERROR: Restore Modem context fail => Factory Reset Modem
INFO: modem_context_factory_reset done
INFO: Use soft secure element for cryptographic functionalities

INFO: ###### ===== LoRa Basics Modem LoRaWAN Class A/C demo application ==== ######

INFO: LoRaWAN version: 01.00.04.01
INFO: LoRa Basics Modem version: 03.02.04
INFO: Modem Initialization
WARN: No valid DevNonce in NVM, use default (0)
 DevNonce = 0
 JoinNonce = 0xff ff ff, NetID = 0xff ff ff
 NbOfReset = 1
 Region = EU868
Modem Load Config :
 Port = 199
Upload_sctr = 0
INFO: Use soft secure element for cryptographic functionalities

INFO: ###### ===== LoRa Basics Modem LoRaWAN Class A/C demo application ==== ######

INFO: LoRaWAN version: 01.00.04.01
INFO: LoRa Basics Modem version: 03.02.04
INFO: ###### ===== BASICS MODEM RESET EVENT ==== ######
Reset count : 1 
INFO: Application parameters:
INFO:   - LoRaWAN uplink Fport = 2
INFO:   - DM report interval   = 60
INFO:   - Confirmed uplink     = No
INFO: LoRaWAN parameters:
DevEUI - (8 bytes):
 70 B3 D5 7E D0 06 36 85
JoinEUI - (8 bytes):
 00 00 00 00 00 00 03 33
Class: SMTC_MODEM_CLASS_A
Region: SMTC_MODEM_REGION_IN_865

 Start a new join sequence in 4 seconds 
DevEUI - (8 bytes):
 70 B3 D5 7E D0 06 36 85
JoinEUI - (8 bytes):
 00 00 00 00 00 00 03 33
DevNonce 0x1

  *************************************
  * Send Payload  HOOK ID = 3
  *************************************
  Tx  LoRa at 4209 ms: freq:865402500, SF11, BW125, len 23 bytes 22 dBm, fcnt_up 0, toa = 824



The module has freeze after this , there is no communication with gateway
